### PR TITLE
Power ups a quite a few other things

### PR DESCRIPTION
**Pull request by bbeck13**
Added camera setup / beginning menu thing addressed
https://github.com/bbeck13/RhythmRunner/issues/179
https://github.com/bbeck13/RhythmRunner/issues/146
I also fixed some shit with the octree
cocaine is red 
acid is green
and of course DMT is purple (if anyone can find a good pill mesh or something that would be dope)

### CODE REVIEW

**Comment by alexottoboni** [comment]
Damn 40 files changed, I'll take a look.

**Comment by alexottoboni** [comment]
QA 👨‍🔬

**Comment by bbeck13** [comment]
thanks for the drugs QA 👍 on the pill models

File: src/game_renderer/GameRenderer.cpp [hunk 1]
```diff
@@ -426,7 +493,18 @@ void GameRenderer::RenderObjects(GLFWwindow* window,
...
+    if (game_state->GetElapsedTicks() % 10 == 0) {
+      glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+    }
+    float r = ((double)rand() / (RAND_MAX));
+    float g = ((double)rand() / (RAND_MAX));
+    float b = ((double)rand() / (RAND_MAX));
+    glClearColor(r, g, b, 1.0);
+  } else {
+    glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+    glClearColor(.2f, .2f, .2f, 1.0f);
```
Comment (alexottoboni) [inline src/game_renderer/GameRenderer.cpp]: Should the order be flipped? ClearColor gets set before clearing?

**Comment by alexottoboni** [comment]
CR 👍  Looks good to me

**Pull request merged**